snmp/tests/compose/data/fortinet-appliance.snmprec

```diff
@@ -1,4 +1,268 @@
 1.3.6.1.2.1.1.1.0|4|Fortinet Appliance dummy device
```

Comment:
The file `fortinet-appliance.snmprec` was already in the repo? 🤔

Reply:
we had an old one when added the `fortinet` vendor profile
it only had the 4 basic lines
similar to [this one](https://github.com/DataDog/integrations-core/blob/de6c9d68b3bfd47b59e1d5d25e5a8f6453f1f18e/snmp/tests/compose/data/fortinet-fortiswitch.snmprec)